RecoEgamma/EgammaTools/python/regressionModifier_cfi.py

```diff
@@ -246,5 +246,16 @@
 
 (run2_egamma_2016 | run2_egamma_2017 | run2_egamma_2018).toReplaceWith(regressionModifier,regressionModifier106XUL)
 
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(regressionModifier106XUL.eleRegs.ecalOnlyMean,
+      rangeMinHighEt = 0.2,
+      rangeMaxHighEt = 2.0
+)
+
+run3_common.toModify(regressionModifier106XUL.phoRegs.ecalOnlyMean,
+      rangeMinHighEt = 0.2,
+      rangeMaxHighEt = 2.0
+)
+
```

Comment:
Could something similar also work? (Please check before implementing, if you decide so)

```suggestion
regressionModifierRun2 = regressionModifier106XUL.clone()

from Configuration.Eras.Modifier_run2_egamma_2016_cff import run2_egamma_2016
from Configuration.Eras.Modifier_run2_egamma_2017_cff import run2_egamma_2017
from Configuration.Eras.Modifier_run2_egamma_2018_cff import run2_egamma_2018

(run2_egamma_2016 | run2_egamma_2017 | run2_egamma_2018).toReplaceWith(regressionModifier,regressionModifierRun2)


regressionModifierRun3 = regressionModifierRun2.clone(
    eleRegs = dict(
        ecalOnlyMean = dict(
            rangeMinHighEt = 0.2,
            rangeMaxHighEt = 2.0
        )
    ),
    phoRegs = dict(
        ecalOnlyMean = dict(
            rangeMinHighEt = 0.2,
            rangeMaxHighEt = 2.0
        )
    )
)

from Configuration.Eras.Modifier_run3_common_cff import run3_common
run3_common.toReplaceWith(regressionModifier,regressionModifierRun3
```

Reply:
what's wrong with the original? except for the fact that there we should just have `run3_common.toModify(regressionModifier.phoRegs.ecalOnlyMean`

Reply:
the original one, ie `regressionModifier` clones `regressionModifier94X`, 
https://github.com/cms-sw/cmssw/blob/65582211de8db178ab9d80da778ac8d466104317/RecoEgamma/EgammaTools/python/regressionModifier_cfi.py#L240
and `regressionModifier94X` is incompatible with Run3 as it uses `EGRegressionModifierV2`, while we need `EGRegressionModifierV3`. 
https://github.com/cms-sw/cmssw/blob/65582211de8db178ab9d80da778ac8d466104317/RecoEgamma/EgammaTools/python/regressionModifier_cfi.py#L183-L185

Reply:
> both have an unintended effect (discussed here also [#37102 (comment)](https://github.com/cms-sw/cmssw/pull/37102#issuecomment-1055948236)), that new parameter values gets propagated to low pT electron regression too.

uhm, what about 
`(run2_egamma_2016 | run2_egamma_2017 | run2_egamma_2018 | run3_common).toReplaceWith(regressionModifier,regressionModifier106XUL.clone())` ?

Reply:
@slava77 I don't think it can work.
For Run3 the regressionModifier has to be further modified: it cannot be the same as for Run2.